src/events/eventTypes.ts

```diff
@@ -102,7 +100,22 @@ export const fromDjsArgs = {
   userUpdate: (oldUser: User | PartialUser, newUser: User) => ({ oldUser, newUser }),
   voiceStateUpdate: (oldState: VoiceState, newState: VoiceState) => ({ oldState, newState }),
   webhookUpdate: (channel: TextChannel) => ({ channel }),
-  interaction: (interaction: Interaction) => ({ interaction }),
+  interactionCreate: (interaction: Interaction) => ({ interaction }),
```

Comment:
Couldn't comment higher up on this object, but this is not up to date with the discord.js ClientEvents (e.g., it's missing the sticker events)

Reply:
All added, ty!